<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demonstration of different area selection modes</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="GraphicsViewEx" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Use the right mouse button to draw new areas and the left mouse button to select existing ones.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>To close a polygonal/lasso selection, click anywhere in the view using the left mouse button.</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="rectangularSelectionBtn">
        <property name="text">
         <string>Rectangular selection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="polygonalSelectionBtn">
        <property name="text">
         <string>Polygonal selection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="lassoSelectionBtn">
        <property name="text">
         <string>Lasso selection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="toggleImageBtn">
        <property name="text">
         <string>Toggle image</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="extractSelectedAreaBtn">
        <property name="text">
         <string>Extract selected area...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphicsViewEx</class>
   <extends>QGraphicsView</extends>
   <header>graphicsviewex.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
